tests/utils/verification/volumeproperties.go

```diff
@@ -109,17 +109,27 @@ func GetFullVolumeName(hostName string, volumeName string) string {
 // shorter name without @datastore suffix.
 func VerifyAttachedStatus(name, hostName, esxName string) bool {
 	log.Printf("Confirming attached status for volume [%s]\n", name)
+	return testAttachedStatus(name, hostName, esxName, esx.RetrieveVMNameFromIP(hostName))
+}
+
+// VerifyAttachedStatus - verify volume is attached and name of the VM attached
```

Comment:
VerifyAttachedStatus => VerifyAttachedStatusByUuid

Moreover, can we add some comments to explain the difference between these 2 functions? When should we use this one in preference to the other, and vice versa?

If one is always preferred than the other, then I'd suggest to make the preferred one as the default one with shorter name/parameter list so that people will use it by default.